### PR TITLE
bug/SSCS-10614: made change as per suggestion on the ticket.

### PR DESCRIPTION
**Pull request by jaberelfasi**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/SSCS-10614

### Change description ###
Missing screenFlow Tag

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by nickwest1 — DISMISSED** [review]

**Review by wmarshid — DISMISSED** [review]

File: src/integrationTest/java/uk/gov/hmcts/reform/sscs/controllers/ServiceHearingsControllerTest.java [hunk 1]
```diff
@@ -204,23 +205,19 @@ void setUp()  {
...
-                .hearingIsLinkedFlag(true)
-                .build();
-        String json = asJsonString(model);
-
         ServiceHearingRequest request = ServiceHearingRequest.builder()
                 .caseId(String.valueOf(CASE_ID))
                 .build();
 
+        String actualJson = ResourceLoader.loadJson("serviceHearingValuesForControllerTest.json");
+        ServiceHearingValues serializedObject = mapper.readValue(actualJson, ServiceHearingValues.class);
```
Comment (HarryH96) [inline src/integrationTest/java/uk/gov/hmcts/reform/sscs/controllers/ServiceHearingsControllerTest.java:213]: What is this object being used for?

Comment (jaberelfasi) [reply src/integrationTest/java/uk/gov/hmcts/reform/sscs/controllers/ServiceHearingsControllerTest.java:213]: not used will be removed. thanks for spotting this

File: src/test/resources/serviceHearingValuesForControllerTest.json [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+      "categoryType":"caseSubType"
+    }
+  ],
+  "duration":30,
+  "hearingPriorityType":"normal",
+  "numberOfPhysicalAttendees":0,
+  "hearingInWelshFlag":false,
+  "caseAdditionalSecurityFlag":false,
+  "facilitiesRequired":[
+
```
Comment (wmarshid) [inline src/test/resources/serviceHearingValuesForControllerTest.json:21]: minor: can we get rid of these trailing newlines please? same comment applies to others in this file.

**Review by wmarshid — APPROVED** [review]

File: src/test/resources/serviceHearingValuesForControllerTest.json (hunk 2 above)
Comment (jaberelfasi) [reply src/test/resources/serviceHearingValuesForControllerTest.json:21]: removed trailing new lines

**Review by wmarshid — APPROVED** [review]

**Review by nickwest1 — APPROVED** [review]

**Pull request merged**